.github/workflows/i18n.yml

```diff
@@ -0,0 +1,26 @@
+name: i18n checks
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+  push:
+    branches:
+      - main
+
+jobs:
+  check-i18n:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.13.1
```

Comment:
Why isn't this the same as above `node-version: 16`

Reply:
That's a good catch!

I've fixed this difference in the "Update Node" PR in Address 

https://github.com/alphagov/di-ipv-cri-address-front/pull/385/files

In this PR I'm using a general Node 16 version everywhere except for Docker and npm install where is a specific node.js version is needed

When I apply that Node 16.17.1 update here, that will fix this issue

Reply:
Or, I could update the PR here 😄